packages/react-server/core/util/ClientCssHelper.js

```diff
@@ -29,49 +31,51 @@ module.exports = {
 			throw new Error("ClientCssHelper.registerPageLoad can't be called server-side");
 		}
 
-		var newCss = pageObject.getHeadStylesheets();
+		return Q.all(PageUtil.standardizeStyles(pageObject.getHeadStylesheets())).then(newCss => {
```

Comment:
Oh, I totally mistook the context here.  Thought we were in `renderMiddleware`.  I'll go gripe over there. 👹